articlestatus/ArticleStatusController.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 var bodyParser = require('body-parser');
-const ArticleStatus = require('../data/models/ArticleStatus');
+const ArticleStatusHelper = require('./ArticleStatusHelper.js');
```

Comment:
Thanks, it feels better to get the data access out of this file and keep the routes clean.